.circleci/config.yml

```diff
@@ -2,13 +2,18 @@
 defaults: &defaults
   docker:
     - image: golang:1.13.4-alpine3.10
-  working_directory: /go/src/github.com/honeydipper/honeydipper
+      environment:
+        - GO111MODULE=auto
 
-restore_deps: &restore_deps
+restore_mod: &restore_mod
   restore_cache:
-    key: gopkg-{{ .Branch }}-{{ checksum "Gopkg.lock" }}
+    key: go-mod-v1-{{ checksum "go.sum" }}
+
+save_mod: &save_mod
+  save_cache:
+    key: go-mod-v1-{{ checksum "go.sum" }}
     paths:
-      - /go/src/github.com/honeydipper/honeydipper/vendor
+      - "/go/pkg/mod"
```

Comment:
Thanks for the contribution!

nit: I _think_ this will work Ok without the quotes?
`restore_deps` vs `restore_mod` shouldn't matter too much, but this should work.